screensaver.bing/resources/skins/default/1080i/default.xml

```diff
@@ -9,6 +9,13 @@
     <animation effect="fade" time="250">WindowOpen</animation>
     <animation effect="fade" time="250">WindowClose</animation>
     <controls>
+        <control type="image">
```

Comment:
I don't think this is the expected behaviour in Leia but likely a regression. Can you please open a bug report in xbmc/xbmc?